### DISCUSSION TITLE
Curso de ADS EAD precisa de computador muito forte?

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#1871] [Discussão]
Eu estou querendo iniciar o curso de ADS(análise e desenvolvimento de sistemas) e estou querendo cursar por EAD, mas eu não achei nada claro na internet sobre quais os requisitos de PC para rodar as ferramentas usadas no curso. Algumas pessoas me disseram que eu precisaria de um PC high end pra poder cursar por EAD, enquanto outras, me disseram que um PC com 2gb de ram já dá pro gasto. Alguem que já fez esse curso,ou está fazendo pode me falar sobre isso pra eu ter uma base melhor de opiniões?

**Comment by brendomorassi** [comment]
Vai depender muito das tecnologias que o curso irá utilizar, mas na grande maioria uma máquina fraquinha no começo ja da pro gasto ... imagina que 512MB de ram vc roda um PHP, Java, C++, etc. Porem no futuro quando quiser trabalhar com docker para criar containers na sua maquina, emular Android, montar um VM (Virtual Machine),etc. Vai sentir a necessidade de algo mais potente. 

Sempre indico quem está começando a se virar com o que tem, o mais importante é aprender, e no futuro pega uma máquina ja pensando na tecnologia que irá utilizar... eu por exemplo comprei um macbook pra publicar na apple e emular ios no xcode...

**Comment by ghost** [comment]
> Vai depender muito das tecnologias que o curso irá utilizar, mas na grande maioria uma máquina fraquinha no começo ja da pro gasto ... imagina que 512MB de ram vc roda um PHP, Java, C++, etc. Porem no futuro quando quiser trabalhar com docker para criar containers na sua maquina, emular Android, montar um VM (Virtual Machine),etc. Vai sentir a necessidade de algo mais potente.
> 
> Sempre indico quem está começando a se virar com o que tem, o mais importante é aprender, e no futuro pega uma máquina ja pensando na tecnologia que irá utilizar... eu por exemplo comprei um macbook pra publicar na apple e emular ios no xcode...

Entendi, mas dá pra fazer o curso inteiro deboa com o pc ruim então?

**Comment by pedrofnsc1** [comment]
> Entendi, mas dá pra fazer o curso inteiro deboa com o pc ruim então?

Amigo atualmente curso ADS e fazer o curso **inteiro** com um pc ruim não da, chega momentos em que você vai precisar usar IDE como o Android Studio não vai nem conseguir abrir dependendo de quantos GB  você tem no seu computador.

para os dois primeiros semestres ao meu ver um pc ruim consegue aguentar o tranco.

**Comment by ghost** [comment]
Bom no estado que eu moro (RN) as únicas faculdades que tem esse curso são a UNP, UNIP, ESTÁCIO e claro a universidade federal que só tem o presencial. tirando a UF que é presencial, essas faculdades que eu citei tem uma nota média no MEC e o curso nem é tão conhecido aqui na região, então, por serem faculdades medianas eu acho que o curso delas não usa uma tecnologia muito forte no ensino, mas não tenho certeza se isso procede. oque vocês acham?

**Comment by pedrofnsc1** [comment]
> Bom no estado que eu moro (RN) as únicas faculdades que tem esse curso são a UNP, UNIP, ESTÁCIO e claro a universidade federal que só tem o presencial. tirando a UF que é presencial, essas faculdades que eu citei tem uma nota média no MEC e o curso nem é tão conhecido aqui na região, então, por serem faculdades medianas eu acho que o curso delas não usa uma tecnologia muito forte no ensino, mas não tenho certeza se isso procede. oque vocês acham?

Eu faço na ADS na UFRN e a grade curricular usa alguns softwares pesados como que eu citei (Android Studio), acho que primeiro é bom você ver qual a grade curricular das faculdades que ofertam o curso e ver a matérias que abordam mais o desenvolvimento de softwares. na grade da UFRN da para usar em quase todas as matérias o VS Code 

Ps: Procure também pelo curso ADS ofertado pelo IFRN

**Comment by pedrofnsc1** [comment]
Olha aqui pelas faculdades que tu citou a maior grade é da ESTÁCIO, mas tem que ver também a qualidade das aulas

**Comment by ghost** [comment]
> Olha aqui pelas faculdades que tu citou a maior grade é da ESTÁCIO, mas tem que ver também a qualidade das aulas

Eu já dei uma olhada na grade de todas, são bem parecidas, mas eu não manjo de qual software pode ser usado em cada matéria. no curso tec que eu fazia no ensino médio nós usavamos o bloco de notas pra programar web(kkkk), então, eu não sei quais softwares são usados em desenvolvimento do nivel de universidades.
